internal/pkg/template/templates/environment/cf.yml

```diff
@@ -284,25 +290,54 @@ Resources:
 {{- else}}
       VpcId: !Ref VPC
 {{- end}}
-  InternalALBHTTPListener:
+  InternalHTTPListener:
     Metadata:
       'aws:copilot:description': 'An internal load balancer listener to route HTTP traffic'
     Type: AWS::ElasticLoadBalancingV2::Listener
     Condition: CreateInternalALB
     Properties:
       DefaultActions:
-        - TargetGroupArn: !Ref DefaultInternalALBHTTPTargetGroup
+        - TargetGroupArn: !Ref DefaultInternalHTTPTargetGroup
           Type: forward
       LoadBalancerArn: !Ref InternalLoadBalancer
       Port: 80
       Protocol: HTTP
+  InternalHTTPSListener:
+    Metadata:
+      'aws:copilot:description': 'An internal load balancer listener to route HTTPS traffic'
+    Type: AWS::ElasticLoadBalancingV2::Listener
+    Condition: ExportInternalHTTPSListener
+    Properties:
+      Certificates:
+{{- if .ImportCertARNs}}
+        - CertificateArn: {{index .ImportCertARNs 0}}
+{{- else}}
+        - CertificateArn: !Ref HTTPSCert
+{{- end}}
```

Comment:
We don't want to use our HTTPSCert for internal load balancers, the certificate has to be private, so no need for the else statement